openedx/core/lib/xblock_utils/__init__.py

```diff
@@ -500,3 +501,32 @@ def xblock_resource_pkg(block):
         return module_name
 
     return module_name.rsplit('.', 1)[0]
+
+
+def is_xblock_aside(usage_key):
+    """
+    Returns True if the given usage key is for an XBlock aside
+
+    Args:
+        usage_key (opaque_keys.edx.keys.UsageKey): A usage key
+
+    Returns:
+        bool: Whether or not the usage key is an aside key type
+    """
+    return isinstance(usage_key, (AsideUsageKeyV1, AsideUsageKeyV2))
+
+
+def get_aside_from_xblock(xblock, aside_type):
+    """
+    Gets an instance of an XBlock aside from the XBlock that it's decorating
+
+    Args:
+        xblock (xblock.core.XBlock): The XBlock that the desired aside is decorating
```

Comment:
Made enough sense to expect an `XBlock` type here. That class inherits from `RuntimeServicesMixin`, and 
this function's only client right now is passing in an `XModuleDescriptor`, which inherits from `XBlock`